### ISSUE TITLE
RFC: shall we add more information to docker_images ?

### ISSUE THREAD

**Issue by PascalPrecht** [machinelabs/machinelabs#814]
Right now the only information available to the client is the `id` and `name` of every docker image we have. While this is enough to get things going, I think it would be useful to also add information about what else is included in a container.

For example, some containers might come with particular libraries pre-installed. That information would be very useful for UIs where people would choose what environment they want to run their labs in.

In fact, since we're working on enabling users to run commands like `npm install` or `pip install` it'd be a waste if people would try to install libraries even though they're already included in the container. Of course, this doesn't really "solve" it as users would still be able to install/override libs, but I do think it'd be useful.

Any thoughts?

**Comment by SamVerschueren** [comment]
I agree that `id` and `name` are quite short and doesn't really mean much. But I'm not sure if we could easily list everything that's preinstalled inside the container. We can also add a link to the Dockerhub page of the image being used which should also give more information. But that might not be really user friendly though.

**Comment by PascalPrecht** [comment]
Yea, I was more thinking of having an ability to show our users more information about the containers inside our platform (e.g. the entry wizard could make great use of that data)

**Comment by PascalPrecht** [comment]
Another thing, that's related to this, is that we might also need to store on what hardware a docker image can run. For example, I remember I once tried to use a docker image that only worked on GPU hardware.

We should provide guidance for our users on what hardware they can use a certain docker image.
